### ISSUE TITLE
Roadmin user doesn't have read privs for admin operations?

### ISSUE THREAD

**Issue by ldangeard-orange** [cloudfoundry/cf-mysql-release#184]
I need REPLICATION CLIENT to monitoring binlogs size with roadmin

```
MariaDB [(none)]> SHOW BINARY LOGS;
ERROR 1227 (42000): Access denied; you need (at least one of) the SUPER, REPLICATION CLIENT privilege(s) for this operation

```
The REPLICATION CLIENT privilege enables the use of the SHOW MASTER STATUS, SHOW SLAVE STATUS, and SHOW BINARY LOGS statements.

**Comment by menicosia** [comment]
Hi @ldangeard-orange this has been addressed in [v36.10.0](https://github.com/cloudfoundry/cf-mysql-release/releases/tag/v36.10.0).

Please let us know if this works for you!

--
  Marco Nicosia
  Product Manager
  Pivotal Software, Inc.

**Comment by ldangeard-orange** [comment]
Thanks !
